ci/playwright_test.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2024 The AI Edge Model Explorer Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import re
+import time
+from playwright.sync_api import Page, expect
+import cv2
+from pathlib import Path
+
+LOCAL_SERVER = "http://127.0.0.1:8080//"
+ROOT_DIR = Path(__file__).parent.parent
+TEST_FILES_DIR = ROOT_DIR / "src/server/test"
+TMP_SCREENSHOT_DIR = ROOT_DIR / "build"
+EXPECTED_SCREENSHOT_DIR = ROOT_DIR / "src/server/screenshots"
```

Comment:
Maybe move the `screenshots` dir under `test` and rename it to `screenshots_golden`, add another dir `chrome-linux` under `screenshots_golden`, and move all the screenshots there? Basically: `test/screehshots_golden/chrome-linux`. This is mainly to match the convention used internally. Also just in case we will add screenshots for other browsers/platforms in the future.

Also maybe consider moving all the test models into a subdir `test_models` under `test`.

Reply:
thx for the suggestions! Done.